packages/cli/src/evaluation.ee/test-runner/test-runner.service.ee.ts

```diff
@@ -360,21 +362,32 @@ export class TestRunnerService {
 	/**
 	 * Get the evaluation set metrics nodes from a workflow.
 	 */
-	static getEvaluationMetricsNodes(workflow: IWorkflowBase) {
+	static getEvaluationNodes(
+		workflow: IWorkflowBase,
+		operation: string,
+		isDefaultOperation = false,
```

Comment:
Nit - Third arg can be an object to avoid passing bare booleans.

Reply:
Why? Can you explain what the object is?

Reply:
I meant like this:

```ts
static getEvaluationNodes(
 workflow: IWorkflowBase,
 operation: string,
 options = { isDefaultOperation: false }
)
```

And then:

```ts
getEvaluationNodes(workflow, 'setMetrics', { isDefaultOperation: true });
```

Reply:
I see thanks. will keep that in mind for future.

Reply:
Updated now.